<?xml version="1.0"?>
<launch>

    <param name="control_ok"        type="bool" value="true" />
    <param name="angle_desired"     type="double" value="1.5708" />    
    
    <param name="rho"               type="double" value="1.0" />
    <param name="theta"             type="double" value="0.017453293" />
    <param name="threshold"         type="double" value="100.0" />

    <param name="odom_dron"         type="str" value="/dji_sdk/odometry" />  
	<param name="imgTopic"          type="str" value="/panel/image/mask" /> 

    <param name="vx_lineal"         type="double" value="0.2" />    <!-- Velocidad lineal -->
    <param name="d_panel_d"         type="double" value="300.0" />   <!-- Ancho en pixeles del panel deseado -->  

    <!-- Publisher -->
    <param name="publish_vel_drone_topic"       type="str" value="/dji_sdk/visual_servoing/vel/drone" />     
    <param name="publish_vel_drone_world_topic" type="str" value="/dji_sdk/visual_servoing/vel/drone_world"/> 


    <!-- ............................... params ........................................ -->
    <group ns="vision_params">
        <rosparam file="$(find vision_matrice100)/cfg/cfg_params_real.yaml" /> 
    </group>
    
   <node pkg="vision_matrice100" type="vs_sn_node" name="vs_sn_node" output="screen"/>

    <arg name="rviz" default="false" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find vision_matrice100)/rviz/vision_matrice100.rviz" />
    </group>

</launch>

